docs/configure

```diff
@@ -478,10 +480,11 @@ ditch_maxmsgsize:
 		and domain = '${quote_mysql:$domain}' \
 		and users.maxmsgsize > 0 \
 		and users.domain_id=domains.domain_id}{${value}K}fail} Kb.
-  local_part_suffix = -*
-  local_part_suffix_optional
+#  local_part_suffix = -*
+#  local_part_suffix_optional
```

Comment:
I'm not sure we should comment these out. Perhaps we should add them to the Debian files instead?